kodi.py

```diff
@@ -91,11 +91,14 @@
 def sanitize_name(media_name):
   # Normalize string
   normalized = unicodedata.normalize('NFKD', media_name).encode('ASCII', 'ignore')
-  # Strip symbols
-  m = RE_NAME_WITH_PARAM.match(normalized)
-  if m:
-    return m.group(1)
-  return media_name
+
+  # Strip things between and including brackets and parenthesis
+  removed_paren = re.sub(r'\([^)]*\)', '', normalized)
+  removed_bracket = re.sub(r'\[[^)]*\]', '', removed_paren)
```

Comment:
Initially I put this in at the beginning because I was having problems matching things like 'The Office' because Kodi has it as 'The Office (US)' and there were some other shows like Archer that Kodi labels as 'Archer (2009)'. I guess we could just add an option for the function that will just strip out the actual characters or the whole thing depending on how you are running it.